implementation/src/test/java/com/teampc/model/test/TestTest.java

```diff
@@ -0,0 +1,125 @@
+package com.teampc.model.test;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Lists;
+import com.teampc.dao.QuestionDAO;
+import com.teampc.model.automation.TestGrader;
+import com.teampc.model.question.Question;
+import com.teampc.model.testtaking.MatchingQuestionResponse;
+import com.teampc.model.testtaking.MultipleChoiceQuestionResponse;
+import com.teampc.model.testtaking.ShortAnswerQuestionResponse;
+import org.junit.Test;
+
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by zarend on 12/9/15.
+ */
+public class TestTest {
+
+   @Test
+   public void testIsOpen() {
+      // TODO
+   }
+
+   @Test
+   public void testTakeTest() {
+      // TODO
+   }
+
+   @Test
+   public void testRemoveQuestion() {
+      // TODO
+   }
+
+   @Test
+   public void testMoveQuestionUp() {
+      com.teampc.model.test.Test test = makeFakeTest();
+      int originalSize = test.getQuestions().size();
+
+      Question<MultipleChoiceQuestionResponse> question1 = new Question<>();
+      question1.setPoints(1);
+      question1.setPrompt("What is the name of the version control tool used in CPE 307?");
+      question1.setType(Question.QuestionType.MULTIPLE_CHOICE);
+
+      int originalIdx = test.getQuestions().indexOf(question1);
+      test.moveQuestionDown(question1);
+      assert(test.getQuestions().size() == originalSize);
```

Comment:
use Assert.AssertTrue instead of assert, this will allow junit to catch the test failure